tests/x509/test_x509.py

```diff
@@ -2610,6 +2622,46 @@ def test_name_constraints(self, nc, backend):
 
     @pytest.mark.requires_backend_interface(interface=RSABackend)
     @pytest.mark.requires_backend_interface(interface=X509Backend)
+    @pytest.mark.parametrize(
+        "add_ext",
+        [
+            x509.TLSFeature([x509.TLSFeatureType.status_request]),
```

Comment:
empty list should be tested I guess

Reply:
This is done (but tested in test_x509_ext)